tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu

```diff
@@ -150,7 +150,7 @@ RUN ${PIP} install --no-cache-dir -U \
     sagemaker==1.58.2 \
     sagemaker-experiments==0.1.13 \
     "sagemaker-tensorflow>=2.2,<2.3" \
-    sagemaker-tensorflow-training==4.0.1 \
+    "sagemaker-tensorflow-training>=20" \
```

Comment:
@laurenyu Is there an upper limit on what version of the toolkit will be compatible with the TensorFlow 2.2.0?

Will it be `>=20,<21` or `>=20,<30`, or will it remain compatible with all future versions?